### DISCUSSION TITLE
Review Process

### DISCUSSION THREAD

**Discussion by bogosj** [bogosj/tesla#40] [General]
Proposal for review process:

I'm comfortable with the current four contributors having merge authority onto main. My feelings on PR reviews is:

Select whomever you see fit as reviewers. Don't gate a merge on **everyone** approving unless that's what you want. I'll typically send a PR to everyone as a "if anyone can look at this, please do". Once some other set of eyes have given it a thumbs up I'd merge.

I don't know if anyone wants something more formal, or something more formal for tagging a release?

@michaelharo @uhthomas @andig

**Comment by uhthomas** [comment]
Thanks for putting this in writing. Sounds sensible to me.

**Reply by andig** [reply]
+1. Personally, I don‘t even own a Tesla and hence depend on others for testing and credentials. If you want to remove me from committers that would be ok, too. Otherwise I‘d be very reluctant to merge myself.
